src/cli/Add.cpp

```diff
@@ -0,0 +1,133 @@
+/*
+ *  Copyright (C) 2017 KeePassXC Team <team@keepassxc.org>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 or (at your option)
+ *  version 3 of the License.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <cstdlib>
+#include <stdio.h>
+
+#include "Add.h"
+
+#include <QCommandLineParser>
+#include <QCoreApplication>
+#include <QStringList>
+#include <QTextStream>
+
+#include "cli/Utils.h"
+#include "core/Database.h"
+#include "core/Entry.h"
+#include "core/Group.h"
+#include "core/PasswordGenerator.h"
+
+Add::Add()
+{
+    this->name = QString("add");
+    this->description = QString("Add an entry to the database.");
```

Comment:
@louib lines 50-52 use `QCoreApplication::translate` on the description strings, wouldn't it generate an incomplete translation if we leave places like line [38](#diff-a12f25f3366e69f7d02a65991053f8eeR38), [90](#diff-a12f25f3366e69f7d02a65991053f8eeR90) and [131](#diff-a12f25f3366e69f7d02a65991053f8eeR131) without it?